src/DependencyInjection/Configuration.php

```diff
@@ -357,6 +356,10 @@ private function addProxyClientSection(ArrayNodeDefinition $rootNode)
                                     ->info('Map of additional headers to include in each ban request.')
                                     ->prototype('scalar')->end()
                                 ->end()
+                                ->enumNode('tag_mode')
```

Comment:
can you please add an `info` node here to explain the tag modes a bit? something like "If you can enable the xkey module in Varnish, use the purgekeys mode for more efficient tag handling"